docs/reference/transform/ecommerce-tutorial.asciidoc

```diff
@@ -42,7 +43,7 @@ Go to *Management* > *Elasticsearch* > *Transforms* in {kib} and use the
 wizard to create a {transform}:
 
 [role="screenshot"]
-image::images/ecommerce-pivot1.jpg["Creating a simple {transform} in {kib}"]
+image::images/ecommerce-pivot1.png["Creating a simple {transform} in {kib}"]
```

Comment:
I would retake this screenshot to use a selection of columns which show off the histogram charts better - specifically try and include some numeric fields. For example:

![image](https://user-images.githubusercontent.com/7405507/87536620-477aa900-c691-11ea-9e17-b28dba178f8b.png)

This selection also includes the numeric `taxless_total_price` and `total_quantity` which you use in the example transform. The distribution chart of the country_iso_codes also looks nice!

Reply:
Thanks @peteharverson I've updated the screenshots with the columns you suggested front and centre